.gitignore

```diff
@@ -4,3 +4,16 @@
 /build/
 /dist/
 /.coverage
+
+# Environments 
+.env 
+.venv 
+env/ 
+venv/ 
+ENV/ 
+env.bak/ 
+venv.bak/
+
+
+#ide
+.idea
```

Comment:
All of this is unrelated to quotations.

If you want it, put it in a separate PR.